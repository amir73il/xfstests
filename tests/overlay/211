#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (C) 2020 CTERA Networks. All Rights Reserved.
#
# FS QA Test 211
#
# Run unionmount testsuite to verify correctness
# with up to two incremental rotated snapshots
#
. ./common/preamble
_begin_fstest auto snapshot remount rotate

_register_cleanup "_unionmount_testsuite_cleanup"

# Import common functions.
. ./common/filter

# real QA test starts here

_supported_fs overlay
_require_overlay_snapshot
_require_unionmount_testsuite sn

_unionmount_testsuite_run --sn=1 --samefs --recycle --verify

# success, all done
echo "Silence is golden"
status=0
exit
