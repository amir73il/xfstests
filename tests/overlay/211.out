QA output created by 211
Silence is golden
